src/main/java/Calculator.java

```diff
@@ -1,17 +1,39 @@
 public class Calculator {
-    public int add(int a, int b){
-        //TODO inser your realisation in method add
+
+    /** Adds two integers and returns the result.
+     * */
+    public int add(int a, int b) {
+        return a + b;
     }
-    public int dif(int a, int b){
-        //TODO inser your realisation in method dif
+
+    /**
+     * Subtracts the second integer from the first and returns the result.
+     */
+    public int dif(int a, int b) {
+        return a - b;
     }
-    public int div(int a, int b){
-        //TODO inser your realisation in method div
+
+    /**
+     * Divides the first integer by the second and returns the result.
+     */
+    public int div(int a, int b) {
+        if (b != 0) {
+            return a / b;
+        } else {
+            throw new IllegalArgumentException("You can't divide by zero");
+        }
     }
-    public int times(int a, int b){
-        //TODO inser your realisation in method times
+
+    /**
+     */
+    public int times(int a, int b) {
+        return a * b;
     }
```

Comment:
You have mixed up the methods

Reply:
No, there it's alright. That error located in my tests 😁